README.md

```diff
@@ -1,5 +1,57 @@
 # stream-freshdesk
 
+A data stream generator for the Freshdesk REST API, written in python 3.
+
+## Quick start
+
+1. Install
+
+    Clone this repository, and then install using setup.py. We recommend using a virtualenv:
+
+    ```bash
+    > virtualenv -p python 3 venv
+    > source venv/bin/activate
+    > python setup.py install
+    ```
+
+2. Get your Freshdesk API Key
+
+    Login to your Freshdesk account, navigate to your profile settings
+    page, and save "Your API Token", you'll need it for the next step.
+
+3. Create the config file
+
+    Create a JSON file containing the api token you just found and
+    the subdomain to your Freshdesk account. The subdomain will take the format
+    `subdomain.freshdesk.com`.
+
+    ```json
+    {"api_token": "your-api-token",
+     "domain": "subdomain"}
+    ```
+
+4. [Optional] Create the initial state file
```

Comment:
is this going to be unnecessary once we have the `default_start_date` configuration param?